PWGHF/D2H/Tasks/taskLcToK0sP.cxx

```diff
@@ -81,14 +83,14 @@ struct HfTaskLcToK0sP {
         continue;
       }
 
-      registry.fill(HIST("hMass"), invMassLcToK0sP(candidate));
+      registry.fill(HIST("hMass"), invMassLcToK0sP(candidate), candidate.pt());
```

Comment:
Use `auto pt = candidate.pt();` and fill with `pt` instead of `candidate.pt()` (comment holds for all histograms below as well)

Reply:
OK, done